src/SFA.DAS.FindEpao.Web/Extensions/StandardVersionsFormatExtensions.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace SFA.DAS.FindEpao.Web.Extensions
+{
+    public static class StandardVersionsFormatExtensions
+    {
+        public static string VersionsArrayToString(string[] standardVersions)
+        {            
+            return standardVersions == null ? string.Empty : string.Join(", ", standardVersions);
```

Comment:
Do you have any tests around the new format extensions?

Reply:
added